### PR TITLE
#406 fix cyrillic rules domain parsing

### PR DESCRIPTION
**Pull request by Mizzick**
#406

### CODE REVIEW

File: Extension/tests/safari-converter/test-safari-converter.js [hunk 1]
```diff
@@ -216,6 +216,20 @@ QUnit.test("Generichide rules", function(assert) {
...
+    var ruleText = 'меил.рф';
+    var ruleTextMarkedDomain = '||меил.рф';
+
+    var result = SafariContentBlockerConverter.convertArray([ ruleText, ruleTextMarkedDomain ]);
+    assert.equal(2, result.convertedCount);
+    assert.equal(0, result.errorsCount);
+    var converted = JSON.parse(result.converted);
+    assert.equal(2, converted.length);
+
+    assert.ok(converted[0].trigger["url-filter"] != ".*");
```
Comment (ameshkov) [inline Extension/tests/safari-converter/test-safari-converter.js]: Why don't you check resulted url-filter value?

Comment (Mizzick) [inline Extension/tests/safari-converter/test-safari-converter.js]: is checked now

**Comment by Mizzick** [comment]
Domain parsing was wrong, I've changed it.

**Comment by ameshkov** [comment]
How did it change the fact that the rule like this `фвыфвыфвфывфывфыв` completely blocks ALL Safari requests?

**Pull request merged**